.vscode/settings.json

```diff
@@ -8,11 +8,14 @@
   "plantuml.exportOutDir": "/uml/lib",
   "cSpell.words": [
     "atqr",
+    "Complish",
```

Comment:
I'll ask you to remove this spelling as it is not correct. The word you're looking for is "Accomplished"